src/qml/QFieldCloudPopup.qml

```diff
@@ -318,7 +322,9 @@ Popup {
             Layout.fillWidth: true
             font: Theme.defaultFont
             text: qsTr('Synchronize')
+            visible: !cloudProjectsModel.layerObserver.deltaFileWrapper.hasError()
             enabled: !!(cloudProjectsModel.currentProjectData && cloudProjectsModel.currentProjectData.CanSync)
+                     && !cloudProjectsModel.layerObserver.deltaFileWrapper.hasError()
```

Comment:
Just thinking out loud, does it make sense to add `deltaFileWrapper.hasError()` to the calculation of `CanSync`?